ltext.py

```diff
@@ -83,10 +83,10 @@ def __repr__(self):
         return 'Span({}, {})'.format(self.start, self.end)
 
     def __eq__(self, other):
-        return self.start == other.start and self.end == other.end
+        return isinstance(other, Span) and self.start == other.start and self.end == other.end
```

Comment:
`other` can be anything that has `start` and `end` property (duck type), not necessarily be a Span.

For example, `SpanTrans` is not a `Span` but implements span interface.

Or it should be modified to:

```python
class Span(object):
    def __eq__(self, other):
        return is_equal(self, other)
```